godtools/App/Share/Data/StringSearcher/StringSearcher.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  StringSearcher.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/7/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+class StringSearcher {
```

Comment:
A couple thoughts on this class.

1. It would be nice if this class could implement an interface so the use cases could utilize dependency inversion and point to an interface.

2. What do you think about making the naming of this class a little more unique?  Maybe something like  IgnoringCaseStringSearcher?  This way if we have more string searcher implementations that implement the interface the naming can be a little more distinct and hint a little more at how searching is accomplished.

Reply:
Hey @rachaelblue let's leave this as is and can come back to creating an interface.  I did say in our last video chat that I'm not too worried about dependency inversion right now.

Once you start implementing the new download language work for that I'd like you to use dependency inversion.